modules/json_form_widget/json_form_widget.module

```diff
@@ -188,4 +119,30 @@ function json_form_widget_entity_delete(EntityInterface $entity) {
   foreach ($files as $file) {
     \Drupal::service('file.usage')->delete($file, 'json_form_widget', $type, $id);
   }
+
+}
+
+/**
+ * Implements hook_preprocess_HOOK() for json_form_widget_file_link.
+ */
+function json_form_widget_preprocess_file_link(array &$variables) {
+  $file = $variables['file'];
+  if (!$file instanceof File) {
+    return;
+  }
+  // Find out if file has json_form_widget usage.
+  $usage = \Drupal::service('file.usage')->listUsage($file);
+  $jfw_usage = FALSE;
+  foreach (array_keys($usage) as $type) {
+    if ($type == 'json_form_widget') {
+      $jfw_usage = TRUE;
+      break;
+    }
+  }
+  // If it does, and the URI does not start with public://, show the full URL as
+  // the link title. Otherwise, show the file name.
+  if ($jfw_usage && (strpos($file->getFileUri(), 'public://') !== 0) && isset($variables['link'])) {
+    unset($variables['file_size']);
```

Comment:
I think we need to remove more than just the file_size. I'm noticing that the parentheses are still visible. See img

![image](https://github.com/user-attachments/assets/af7a8c73-7d80-48f6-8607-a20428155d32)

Reply:
Yeah I should have documented this as a known issue, I'll update the description.

This is Claro theme's fault, it provides a twig template for this widget that does not surround the parentheses with a conditional to see if the file size variable is set. Let's discuss what the best way to address would be.

Reply:
Not a bad idea to request the file size from the HEAD.